tools/issue-labeler/src/IssueLabelerService/Answers/OpenAiAnswerService.cs

```diff
@@ -208,8 +209,20 @@ private async Task RetrieveAndAggregateSubqueryResults(
                 { "PrintableDocs", printableDocs },
                 { "PrintableIssues", printableIssues }
             };
-            
-            return (printableIssues, printableDocs, replacementsUserPrompt);
+
+            return replacementsUserPrompt ;
+        }
+
+        class IssueIdComparer : IEqualityComparer<Issue>
```

Comment:
```suggestion
        private class IssueIdComparer : IEqualityComparer<Issue>
```

We'll always want to use explicit visibility scoping, even though `private` is the default in this case.